src/app/zap-templates/templates/app/cluster-objects-src.zapt

```diff
@@ -7,6 +7,41 @@ namespace app {
 namespace Clusters {
 
 namespace detail {
+
+CHIP_ERROR FlightCheckDecodeAndEnterStruct(TLV::TLVReader & reader, TLV::TLVType & outer)
+{
+    TLV::TLVReader temp_reader;
+
+    // Make a copy of the struct reader to do pre-checks.
+    temp_reader.Init(reader);
+
+    // Ensure we have a single struct and that it's properly bounded.
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    VerifyOrReturnError(TLV::kTLVType_Structure == temp_reader.GetType(), CHIP_ERROR_WRONG_TLV_TYPE);
+    ReturnErrorOnFailure(temp_reader.EnterContainer(outer));
+    while ((err = temp_reader.Next()) == CHIP_NO_ERROR)
+    {
+        if (!TLV::IsContextTag(temp_reader.GetTag()))
```

Comment:
It already merged, but this does nothing it seems ... we can remove later (probably trivial size change, if not already removed by the compiler)